src/scenes/UserProfile/UserProfile.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'react-router-dom'
+
+import NotFound from 'scenes/NotFound'
+import Loading from 'components/Loading'
+import Card, { CardText } from 'components/Card'
+
+import { getAllPersonalData } from './queries'
+
+const UserProfile = props => {
+  if (!props.data) {
+    props.history.push('/auth/login')
+  }
```

Comment:
You shouldn't need this condition if you don't skip the request (see other review comment for explanation)